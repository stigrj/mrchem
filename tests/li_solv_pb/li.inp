{
"world_prec": 1.0e-3,
"world_size": 5,
"MPI": {
  "numerically_exact": true
},
"Molecule": {
  "charge": 1,
  "coords": "Li 0.0 0.0 0.0"
},
"WaveFunction": {
  "method": "pbe0",
  "environment": "pcm"
},
"PCM": {
  "SCRF": {
    "kain": 5,
    "max_iter": 100,
    "dynamic_thrs": false,
    "optimizer": "potential",
    "run_pb": true
  },
  "Cavity": {
    "spheres": "0 4.0 1.0 0.0 0.5"
  },
  "Permittivity": {
    "epsilon_in": 1.0,
    "epsilon_out": 2.0,
    "formulation": "exponential"
  },
  "Debye-Huckel_screening": {
    "kappa_out": 2.0,
    "ion_radius": 3.5,
    "ion_width": 3.0
  }
  
},
"SCF": {
  "run": false,
  "guess_type": "sad_dz"
}
}
