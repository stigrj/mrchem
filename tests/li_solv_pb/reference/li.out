                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                                                                     ***
***          __  __ ____   ____ _                                       ***
***         |  \/  |  _ \ / ___| |__   ___ _ __ ___                     ***
***         | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \                    ***
***         | |  | |  _ <| |___| | | |  __/ | | | | |                   ***
***         |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|                   ***
***                                                                     ***
***         VERSION            1.2.0-alpha                              ***
***                                                                     ***
***         Git branch         pb-solvation                             ***
***         Git commit hash    32bcdd958a32a50f6e75-dirty               ***
***         Git commit author  Gabriel Gerez                            ***
***         Git commit date    Tue Apr 18 11:35:36 2023 +0200           ***
***                                                                     ***
***         Contact: luca.frediani@uit.no                               ***
***                                                                     ***
***         Radovan Bast            Magnar Bjorgve                      ***
***         Roberto Di Remigio      Antoine Durdek                      ***
***         Luca Frediani           Gabriel Gerez                       ***
***         Stig Rune Jensen        Jonas Juselius                      ***
***         Rune Monstad            Peter Wind                          ***
***                                                                     ***
***************************************************************************

---------------------------------------------------------------------------

 MPI processes           :       (no bank)                               1
 OpenMP threads          :                                              12
 Total cores             :                                              12
                                                                           
---------------------------------------------------------------------------

XCFun DFT library Copyright 2009-2020 Ulf Ekstrom and contributors.
See http://dftlibs.org/xcfun/ for more information.

This is free software; see the source code for copying conditions.
There is ABSOLUTELY NO WARRANTY; not even for MERCHANTABILITY or
FITNESS FOR A PARTICULAR PURPOSE. For details see the documentation.
Scientific users of this library should cite
U. Ekstrom, L. Visscher, R. Bast, A. J. Thorvaldsen and K. Ruud;
J.Chem.Theor.Comp. 2010, DOI: 10.1021/ct100117s

---------------------------------------------------------------------------

 MRCPP version         : 1.4.1
 Git branch            : HEAD
 Git commit hash       : 75d41879b1908a94a452
 Git commit author     : Stig Rune Jensen
 Git commit date       : Thu Jan 6 11:38:53 2022 +0100

 Linear algebra        : EIGEN v3.4.0
 Parallelization       : OpenMP (12 threads)

---------------------------------------------------------------------------


                                                                           
===========================================================================
                         MultiResolution Analysis
---------------------------------------------------------------------------
 polynomial order      : 5
 polynomial type       : Interpolating
---------------------------------------------------------------------------
 total boxes           : 8
 boxes                 : [          2           2           2 ]
 unit lengths          : [   16.00000    16.00000    16.00000 ]
 scaling factor        : [    1.00000     1.00000     1.00000 ]
 lower bounds          : [  -16.00000   -16.00000   -16.00000 ]
 upper bounds          : [   16.00000    16.00000    16.00000 ]
 total length          : [   32.00000    32.00000    32.00000 ]
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                        Initializing Molecule                        ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                             Solvation Cavity
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Dielectric constant     :            (in)                        1.000000
                         :           (out)                        2.000000
---------------------------------------------------------------------------
    N      R_0  Alpha Beta Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   4.0000  1.00  0.00  0.50  ->  4.0000  0.000000  0.000000  0.000000
===========================================================================


===========================================================================
              Square of the Debye-Huckel screening parameter
---------------------------------------------------------------------------
 Formulation             :                                     exponential
 Screening function value:            (in)                        0.000000
                         :           (out)                        2.000000
---------------------------------------------------------------------------
    N      R_0  Alpha Beta Sigma      Radius         x         y         z
---------------------------------------------------------------------------
    0   7.5000  1.00  0.00  3.00  ->  7.5000  0.000000  0.000000  0.000000
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                Computing Initial Guess Wavefunction                 ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial orbitals
 Method                  : Diagonalize SAD Hamiltonian
 Precision               : 1.00000e-03
 Screening               : 1.20000e+01 StdDev
 Restricted              : True
 Functional              : LDA (SVWN5)
 AO basis                : Hydrogenic orbitals
 Zeta quality            : 2
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                            Molecular Orbitals
---------------------------------------------------------------------------
 Alpha electrons         :                                               1
 Beta electrons          :                                               1
 Total electrons         :                                               2
---------------------------------------------------------------------------
    n  Occ Spin          :                                            Norm
---------------------------------------------------------------------------
    0    2    p          :                              1.000000000000e+00
===========================================================================


~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Calculation             : Compute initial energy
 Method                  : DFT (PBE0)
 Relativity              : None
 Environment             : PCM
 External fields         : None
 Precision               : 1.00000e-03
 Localization            : Off
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


===========================================================================
                        Molecular Energy (initial)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.675564301900
 E-N energy              :            (au)                -16.530755287695
 Coulomb energy          :            (au)                  3.332799383894
 Exchange energy         :            (au)                 -0.416484495318
 X-C energy              :            (au)                 -1.283772997761
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.222649094979
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.222649094979e+00
                         :      (kcal/mol)             -4.532280734932e+03
                         :        (kJ/mol)             -1.896306259496e+04
                         :            (eV)             -1.965382942427e+02
===========================================================================


===========================================================================
                        Orbital Energies (initial)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -2.332093141176
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.664186282352
===========================================================================


                                                                           
***************************************************************************
***                                                                     ***
***                    Printing Molecular Properties                    ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
===========================================================================
                                 Molecule
---------------------------------------------------------------------------
 Charge                  :                                               1
 Multiplicity            :                                               1
---------------------------------------------------------------------------
    N    Atom            :               x               y               z
---------------------------------------------------------------------------
    0      Li            :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Center of mass          :        0.000000        0.000000        0.000000
===========================================================================


===========================================================================
                         Molecular Energy (final)
---------------------------------------------------------------------------
 Kinetic energy          :            (au)                  7.675564301900
 E-N energy              :            (au)                -16.530755287695
 Coulomb energy          :            (au)                  3.332799383894
 Exchange energy         :            (au)                 -0.416484495318
 X-C energy              :            (au)                 -1.283772997761
 N-N energy              :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Electronic energy       :            (au)                 -7.222649094979
 Nuclear energy          :            (au)                  0.000000000000
---------------------------------------------------------------------------
 Total energy            :            (au)             -7.222649094979e+00
                         :      (kcal/mol)             -4.532280734932e+03
                         :        (kJ/mol)             -1.896306259496e+04
                         :            (eV)             -1.965382942427e+02
===========================================================================


===========================================================================
                         Orbital Energies (final)
---------------------------------------------------------------------------
    n  Occ Spin          :                                         Epsilon
---------------------------------------------------------------------------
    0    2    p          :            (au)                 -2.332093141176
---------------------------------------------------------------------------
 Sum occupied            :            (au)                 -4.664186282352
===========================================================================


===========================================================================
                           Dipole Moment (dip-1)
---------------------------------------------------------------------------
 r_O                     :        0.000000        0.000000        0.000000
---------------------------------------------------------------------------
 Electronic vector       :       -0.000000       -0.000000       -0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Nuclear vector          :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
---------------------------------------------------------------------------
 Total vector            :        0.000000        0.000000        0.000000
 Magnitude               :            (au)                        0.000000
                         :         (Debye)                        0.000000
===========================================================================


                                                                           
                                                                           
***************************************************************************
***                                                                     ***
***                            Exiting MRChem                           ***
***                                                                     ***
***                       Wall time :  0h  0m  6s                       ***
***                                                                     ***
***************************************************************************
                                                                           
                                                                           
